apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerParameterTests.cs

```diff
@@ -31,6 +31,8 @@ public static IEnumerable<object[]> GetDbTypeConversions()
             yield return new object[] { SpannerDbType.Timestamp, DbType.DateTime, true };
             yield return new object[] { SpannerDbType.Float64, DbType.Double, true };
             yield return new object[] { SpannerDbType.Int64, DbType.Int64, true };
+            // There is no DbType that will map automatically to SpannerDbType.PgOid.
+            yield return new object[] { SpannerDbType.PgOid, DbType.Int64, false };
```

Comment:
nit: Can you move these to after line 41 and update the comment in line 39? I think it's clearer if we have all the types that are not mapped automatically together.

Reply:
That makes sense. All done.